File: q1281.py
class Solution(object):
    def subtractProductAndSum(self, n):
        p=1
        s=0
        
        while n!= 0:
            digit = n%10
            p=p*digit
            s=s+digit
            n=n//10
        return p-s

        
       
        

File: Q1281.c
int subtractProductAndSum(int n) {
    int p=1;
    int sum =0;
    int digit;
    while (n!=0){
        digit=n%10;
        p=p*digit;
        sum = sum+digit;
        n=n/10;
    }
    return (p-sum);
    
    
}
